recipes/datadistillr/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "datadistillr" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5f00109f3839eeffe4f705d6bccf53feb42bffbe7f2a76c3a2ec643c5d33d49d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - pandas
+    - requests
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - datadistillr
+    - datadistillr.tests
+
+about:
+  home: https://github.com/datadistillr/datadistillr-python-sdk
+  license: Apache-2.0
+  license_family: Apache
+  license_file: {{ environ["RECIPE_DIR"] }}/LICENSE
```

This is fine just to do:
```suggestion
  license_file: LICENSE
```

The problem is the upstream source distribution on PyPI doesn't include the `LICENSE` file. So either that needs to be added and a new release made, or you can add it here in the `datadistillr` directory (next to `meta.yaml`).

---

While the path here is fine, you either need a new upstream release that includes `LICENSE` in the source distribution (by adding a `MANIFEST.in` for your project) or you need to manually copy and add the `LICENSE` to this PR. The former is preferred.

---

@dopplershift Is there an example `MANIFEST.in` that you could point me to so that I can get this right?  
Thanks for all your help!

---

There't the [python packaging docs](https://packaging.python.org/en/latest/guides/using-manifest-in/) that describe it. I think what you'd really need in the `MANIFEST.in` is:
```
include LICENSE
```